include/mbedtls/cipher.h

```diff
@@ -71,16 +71,16 @@ extern "C" {
 /**
  * \brief     Supported cipher types.
  *
- * \warning   DES is considered weak cipher and its use
- *            constitutes a security risk. Arm recommends considering stronger
+ * \warning   DES/3DES are considered weak ciphers and their use
+ *            constitutes a security risk. We recommend considering stronger
  *            ciphers instead.
```

Comment:
Sorry, I don't follow what you're suggesting?

Reply:
sorry, I was wondering if it would be a stronger (better?) warning to say "We don't recommend using them." rather than "We recommend considering stronger ciphers instead."

i.e. "don't" rather than "consider doing something else"

Reply:
Possibly, but I think the message is clear enough. In the interests of avoiding churn, I won't change it in this PR, but maybe we could strengthen it when DES is formally retired by NIST.